docs/README.md

```diff
@@ -861,7 +861,7 @@ curl --output db.tar.gz https://iota.lukaseder.de/downloads/db.tar.gz
 ```
 Unpack it:
 ```sh
-tar zxvf db.tar.gz
+mkdir -p db && pv db.tar.gz | tar xzf - -C ./
```

Comment:
Actually, I think this file README.md can be removed. I started using readthedocs. 

Here's that page:
http://iri-playbook.readthedocs.io/en/docs/faq.html#where-can-i-get-a-fully-synced-database-to-help-kick-start-my-node

The file can be edited in the 'docs' branch (just branch off it) https://github.com/nuriel77/iri-playbook/blob/docs/docs/faq.rst

Reply:
Going to edit "docs" branch. Are you going to merge docs into master?